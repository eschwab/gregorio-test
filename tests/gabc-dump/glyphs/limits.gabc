%notes: limit cases
initial-style: 0;
%%
a_.'(a_.') b_.'(b_.') c_.'(c_.') k_.'(k_.') l_.'(l_.') m_.'(m_.'Z)
a/dc(a/dc) a//dc(a//dc) a dc(a dcZ)
a_0.'(a_0.') b_0.'(b_0.') c_0.'(c_0.') k_0.'(k_0.') l_0.'(l_0.') m_0.'(m_0.'Z)
