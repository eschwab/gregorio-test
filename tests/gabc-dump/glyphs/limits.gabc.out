=====================================================================
 SCORE INFOS
=====================================================================
   number_of_voices          1


=====================================================================
 VOICES INFOS
=====================================================================
  Voice 1
   initial_key               5 (c3)


=====================================================================
 SCORE
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 _
---------------------------------------------------------------------
     character                 .
---------------------------------------------------------------------
     character                 '
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  a
         shape                  1 (S_PUNCTUM)
         signs                  6 (_V_EPISEMUS_PUNCTUM_MORA)
         h_episemus_type        2 (H_ALONE)
         h_episemus_top_note    a
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     character                 _
---------------------------------------------------------------------
     character                 .
---------------------------------------------------------------------
     character                 '
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  b
         shape                  1 (S_PUNCTUM)
         signs                  6 (_V_EPISEMUS_PUNCTUM_MORA)
         h_episemus_type        2 (H_ALONE)
         h_episemus_top_note    b
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 c
---------------------------------------------------------------------
     character                 _
---------------------------------------------------------------------
     character                 .
---------------------------------------------------------------------
     character                 '
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  c
         shape                  1 (S_PUNCTUM)
         signs                  6 (_V_EPISEMUS_PUNCTUM_MORA)
         h_episemus_type        2 (H_ALONE)
         h_episemus_top_note    c
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 k
---------------------------------------------------------------------
     character                 _
---------------------------------------------------------------------
     character                 .
---------------------------------------------------------------------
     character                 '
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  k
         shape                  1 (S_PUNCTUM)
         signs                  6 (_V_EPISEMUS_PUNCTUM_MORA)
         h_episemus_type        2 (H_ALONE)
         h_episemus_top_note    k
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 l
---------------------------------------------------------------------
     character                 _
---------------------------------------------------------------------
     character                 .
---------------------------------------------------------------------
     character                 '
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  l
         shape                  1 (S_PUNCTUM)
         signs                  6 (_V_EPISEMUS_PUNCTUM_MORA)
         h_episemus_type        2 (H_ALONE)
         h_episemus_top_note    l
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 m
---------------------------------------------------------------------
     character                 _
---------------------------------------------------------------------
     character                 .
---------------------------------------------------------------------
     character                 '
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  m
         shape                  1 (S_PUNCTUM)
         signs                  6 (_V_EPISEMUS_PUNCTUM_MORA)
         h_episemus_type        2 (H_ALONE)
         h_episemus_top_note    m
---------------------------------------------------------------------
     type                    8 (GRE_END_OF_LINE)
     element_type            13 (GRE_END_OF_PAR)
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 /
---------------------------------------------------------------------
     character                 d
---------------------------------------------------------------------
     character                 c
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  a
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            19 (G_FLEXA)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  d
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  c
         shape                  1 (S_PUNCTUM)
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 /
---------------------------------------------------------------------
     character                 /
---------------------------------------------------------------------
     character                 d
---------------------------------------------------------------------
     character                 c
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  a
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    9 (GRE_SPACE)
     element_type            53 (SP_LARGER_SPACE)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            19 (G_FLEXA)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  d
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  c
         shape                  1 (S_PUNCTUM)
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                  
---------------------------------------------------------------------
     character                 d
---------------------------------------------------------------------
     character                 c
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  a
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    9 (GRE_SPACE)
     element_type            54 (SP_GLYPH_SPACE)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            19 (G_FLEXA)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  d
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  c
         shape                  1 (S_PUNCTUM)
=====================================================================
